src/locales/en-US/commands.json

```diff
@@ -1701,5 +1701,15 @@
     "verified": "Verified account",
     "yes": "Yes",
     "no": "No"
+  },
+  "eclipseplugin": {
+    "commandDescription": "Search for plugins on the Eclipse Plugin Marketplace",
+    "commandUsage": "<plugin name>",
+    "marketplace": "Eclipse Plugin Marketplace",
+    "by": "by [{{name}}]({{homepage}})",
+    "installs": "{{count}} installs",
+    "lastUpdate": "`{{version}}` was published {{ago}}",
+    "license": "License",
+    "install": "Install using Eclipse Marketplace Client"
```

Comment:
```suggestion
    "install": "Drag this link to your Eclipse workspace to install"
```

Reply:
Does it actually work?

Reply:
It's the same link that the website tells you to drag into eclipse, so yeah, it should work.

Reply:
"it should work" famous last words